packages/L10n/src/L10n.js

```diff
@@ -5,15 +5,17 @@ import L10nContext from "./L10nContext";
 
 const propTypes = {
   locale: PropTypes.string,
+  Locales: PropTypes.node,
   children: PropTypes.node.isRequired,
 };
 
 const defaultProps = {
   locale: "en",
+  Locales: null,
 };
 
 export default function L10n(props) {
-  const value = React.useMemo(() => getI18nObject(props.locale), [props.locale]);
+  const value = React.useMemo(() => getI18nObject(props.locale, props.Locales), [props.locale]);
```

Comment:
While unlikely to cause issues, better to add the new prop to the dependencies of `useMemo` as well